clients/src/main/java/org/apache/kafka/clients/consumer/internals/DefaultBackgroundThread.java

```diff
@@ -166,12 +171,18 @@ public DefaultBackgroundThread(final Time time,
                         config,
                         coordinatorRequestManager,
                         groupState);
-            }
-
-            this.requestManagers = new RequestManagers(
+                this.requestManagers = new RequestManagers(
                     offsetsRequestManager,
-                    Optional.ofNullable(coordinatorRequestManager),
```

Comment:
Using this Optional.ofNullable here was introduced to simplify the code and have a single call to the RequestManagers constructor vs 2 calls (if-else). Is there a reason to go back to this longer version?

Reply:
I think it was a merge conflict thing... reverting it!

Reply:
kk updated!